apps/mull-api/src/app/media/media.service.ts

```diff
@@ -54,4 +55,30 @@ export class MediaService {
   async getMedia(mediaId: number): Promise<Media> {
     return await this.mediaRepository.findOne(mediaId);
   }
+
+  deleteStoredFile(media: MediaInput): boolean {
+    unlinkSync(join(process.cwd(), `/apps/mull-api/uploads/${media.id}.${media.mediaType}`));
+    return true;
+  }
```

Comment:
I'd prefer to have this and any helper functions at the bottom of this file after the functions that call them. Makes for easier reading